homework-1/src/repository/Calculator.java

```diff
@@ -0,0 +1,50 @@
+package repository;
+
+import java.util.Scanner;
+
+public class Calculator {
+	/*
+	 * Functia rezolvare primeste doua numere si o operatie si in functie de
+	 * operatie, alege ce face cu acele doua numere
+	 */
+	public double rezolvare(double numar, char operatie, double numar2) {
+		double raspuns = 0;
+		switch (operatie) {
+		case '+':
+			raspuns = numar + numar2;
+			break;
+
+		case '-':
+			raspuns = numar - numar2;
+			break;
+
+		case '*':
+			raspuns = numar * numar2;
+			break;
+
+		case '/':
+			raspuns = numar / numar2;
+			break;
+
+		default:
+			System.out.printf("You have entered wrong operator");
+		}
+		return raspuns;
+	}
+	/* Functia "ruleaza" citeste 2 numere si un operator 
+	 * de la tastatura si afiseaza pe ecran rezultatul operatiei
+	 */
+	public void ruleaza() {

```

Comment:
not ok - Calculator should be used from outside, ruleaza is not a behaviour of calculator